logging-manager.coffee

```diff
@@ -1,14 +1,33 @@
 bunyan = require('bunyan')
+request = require('request')
 
 module.exports = Logger =
 	initialize: (name) ->
-		level = process.env['LOG_LEVEL'] or "debug"
+		@defaultLevel = process.env['LOG_LEVEL'] or if process.env["NODE_ENV"] == 'production' then "warn" else  "debug"
```

Comment:
do a toLowercase on the `process.env["NODE_ENV"]`?

Reply:
that is fine. what do you think about using an env var that is google cloud specific or do you think it doesn't matter?

Reply:
we can set LOG_LEVEL on all apps in google cloud easily enough so no need todo anything too custom imo

Reply:
what i meant was instead of enabling this with NODE_ENV=production, which would apply everywhere, use a different env flag that only enables the check if it is running in google cloud

Reply:
I think this is good for the moment, lets see how how the devs feel about it in action.